model/Image.php

```diff
@@ -711,7 +711,7 @@ public function cacheFilename($format) {
 		array_shift($args);
 		$folder = $this->ParentID ? $this->Parent()->Filename : ASSETS_DIR . "/";
 
-		$format = $format . base64_encode(json_encode($args, JSON_NUMERIC_CHECK));
+		$format = $format . base64_encode(json_encode(array_map('strval', $args)));
```

Comment:
Don't know if this is related but @JorisDebonnet noted a problem with base64_encode vs base64_urlencode and fixed it up in [a PR](https://github.com/silverstripe/silverstripe-framework/pull/4328) that's awaiting review.